### PR TITLE
Add basic Stargate wasmd scripts

### PR DESCRIPTION
**Pull request by willclarktech**
Part of #515

### CODE REVIEW

File: scripts/wasmd/init.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+echo "Waiting for blockchain and REST server to be available ..."
+timeout 60 bash -c "until curl -s http://localhost:1319/node_info > /dev/null; do sleep 0.5; done"
+# The chain is unreliable in the first second of its existence (https://gist.github.com/webmaster128/8175692d4af5e6c572fddda7a9ef437c)
+sleep 1
+echo "Waiting for height to be > 1 ..."
+sleep 10
+# timeout 20 bash -c "until test $(curl -s localhost:1319/blocks/latest | jq -r '.block.header.height // 0') -gt 1; do sleep 0.5; done"
```
Comment (willclarktech) [inline scripts/wasmd/init.sh:11]: @webmaster128 I think I'm doing something stupid here, but I can't get this line to work with the `timeout`.

File: scripts/wasmd/generate_template.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    -e 's/^timeout_precommit =.*$/timeout_precommit = "300ms"/' \
+    -e 's/^timeout_precommit_delta =.*$/timeout_precommit_delta = "100ms"/' \
+    -e 's/^timeout_commit =.*$/timeout_commit = "1s"/' \
+    "template/.simapp/config/config.toml"
+
+  # Custom settings app.toml
+  sed -i "" \
+    -e 's/^enable =.*$/enable = true/' \
+    -e 's/^enabled-unsafe-cors =.*$/enabled-unsafe-cors = true/' \
+    "template/.simapp/config/app.toml"
```
Comment (webmaster128) [inline scripts/wasmd/generate_template.sh:65]: `.simapp`?

File: scripts/wasmd/generate_template.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  sed -i "" \
+    -e 's/^cors_allowed_origins =.*$/cors_allowed_origins = ["*"]/' \
+    -e 's/^timeout_propose =.*$/timeout_propose = "300ms"/' \
+    -e 's/^timeout_propose_delta =.*$/timeout_propose_delta = "100ms"/' \
+    -e 's/^timeout_prevote =.*$/timeout_prevote = "300ms"/' \
+    -e 's/^timeout_prevote_delta =.*$/timeout_prevote_delta = "100ms"/' \
+    -e 's/^timeout_precommit =.*$/timeout_precommit = "300ms"/' \
+    -e 's/^timeout_precommit_delta =.*$/timeout_precommit_delta = "100ms"/' \
+    -e 's/^timeout_commit =.*$/timeout_commit = "1s"/' \
+    "template/.simapp/config/config.toml"
```
Comment (webmaster128) [inline scripts/wasmd/generate_template.sh:59]: `.simapp`?

File: scripts/wasmd/deploy_cw3.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env node
```
Comment (webmaster128) [inline scripts/wasmd/deploy_cw3.js:1]: Can you delete the deployment scripts (and contracts) for now? They cannot work before https://github.com/cosmos/cosmjs/issues/491

File: scripts/wasmd/generate_addresses.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  {
+    mnemonic: "degree tackle suggest window test behind mesh extra cover prepare oak script",
+    accountNumbers: [0],
+  },
+
+  {
+    mnemonic: "example indicate trick cereal hub fix civil host kiss version bird dash",
+    accountNumbers: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+  },
+];
```
Comment (webmaster128) [inline scripts/wasmd/generate_addresses.js:34]: Nice, thanks

File: scripts/wasmd/init.sh (hunk 1 above)
Comment (webmaster128) [reply scripts/wasmd/init.sh:11]: Hmm, `[`, the nice syntax for `test` always needs a space before the end. Maybe an issue here? 

```suggestion
# timeout 20 bash -c "until [ $(curl -s localhost:1319/blocks/latest | jq -r '.block.header.height // 0') -gt 1 ]; do sleep 0.5; done"
```

File: scripts/wasmd/manual_start.sh [hunk 6]
```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```
Comment (webmaster128) [inline scripts/wasmd/manual_start.sh:1]: Have you ever used this script? If not, let's delete it here at least for the new chain.

**Review by webmaster128 — COMMENTED** [review]
Could you add the new ports to `HACKING.md`

File: scripts/wasmd/init.sh (hunk 1 above)
Comment (willclarktech) [reply scripts/wasmd/init.sh]: I tried that syntax before `test`.

Comment (webmaster128) [reply scripts/wasmd/init.sh]: Ahh, `$(curl -s localhost:1319/blocks/latest | jq -r '.block.header.height // 0')` is executed only once and then the following runs:

```
timeout 20 bash -c "until test 1 -gt 1; do sleep 0.5; done"
```

Comment (webmaster128) [reply scripts/wasmd/init.sh]: Found a way to fix the script. I just wonder why we need min height 2 instead of min height 1?

**Pull request closed without merging**